src/pouchdb.js

```diff
@@ -7,13 +7,28 @@ import PouchDBFind from 'pouchdb-find'
 PouchDB.plugin(PouchDBQuickSearch)
 PouchDB.plugin(PouchDBFind)
 
-const db = new PouchDB({
+const pouchdbOptions = {
     name: 'webmemex',
-    auto_compaction: true,
-})
+    autocomplete: true,
+}
 
-export default db
+function initDB() {
+    return PouchDB(pouchdbOptions)
+}
 
+function initTestDB() {
+    const PouchDB = require('pouchdb-node')
+    return PouchDB({
+        name: 'testdb',
+        auto_compaction: true,
```

Comment:
I'd prefer 
```
...pouchdbOptions,
name: 'testdb',
```
so that any changes in the options would not have to be repeated here.